index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>modern art</title>
+    <meta charset="utf-8"/>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"/>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+    <script language=”JavaScript” type=”text/javascript” src=”jQuery.js”></script>
+    <script language=”JavaScript” type=”text/javascript” src=”script.js”></script>
+  </head>
+  <body>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
+    <blockquote class="blockquote text-center quote">
+      <p class="mb-0 quote-text">"Every artist was first an amateur"</p>
+      <footer class="blockquote-footer">Ralph Waldo Emerson</footer>

```

Comment:
czemu tag footer, skoro to ewidentnie nie jest footer na stronie?

Reply:
![saaswsas](https://user-images.githubusercontent.com/37559519/38204991-436da85e-36a5-11e8-8aef-9503bb7e612f.png)

Reply:
robiłam wg Bootstrapa